Makefile

```diff
@@ -50,38 +51,38 @@ checks:
 
 .PHONY: tests
 tests:
-	$(DOCKER_COMPOSE) run --rm --service-ports web /bin/bash -c \
+	$(DOCKER_COMPOSE) run --rm --service-ports $(SERVICE_NAME) /bin/bash -c \
 	"pytest -s --pdbcls=IPython.terminal.debugger:Pdb -vv"
 
 all-tests: checks tests
 
 fix-imports:
-	$(DOCKER_COMPOSE) run --rm web /bin/bash -c "isort -y"
+	$(DOCKER_COMPOSE) run --rm $(SERVICE_NAME) /bin/bash -c "isort -y"
 
 .PHONY: run
 run:
-	$(DOCKER_COMPOSE) run --rm --service-ports web
+	$(DOCKER_COMPOSE) run --rm --service-ports app
```

Comment:
needs `app` to be replaced with `$(SERVICE_NAME)`

Reply:
(weird find and replace)